Cargo.toml

```diff
@@ -29,4 +29,5 @@ tracing = "0.1"
 tracing-subscriber = { version = "0.3", features = ["env-filter", "fmt", "json"] }
 
 [profile.release]
-debug = true
+debug = 1
+#lto = true # Enable link-time optimization
```

Comment:
Disabling this for now because it has errors with `rocksdb 0.18` (due to `bz_internal_error` duplication when linking). It is already fixed ([more info](https://github.com/rust-rocksdb/rust-rocksdb/issues/609)) but need to update https://github.com/sourmash-bio/sourmash/pull/2230 to use the newer rocksdb version.